spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -664,7 +664,7 @@ def getRegistrationStackSh(saltEnabled):
 	        apt-get purge salt-common
 	        rm -rf /etc/salt/minion.d/
         fi
-        apt-get --yes install $A_MISSING
+        apt-get --yes install --no-install-recommends $A_MISSING
```

Comment:
This prevents getting extra "recommended" dependencies: `debconf-utils`, `python3-tz` and `python3-croniter`. I got the same test results with or without them. Even if we discover that they are really needed, then we should make them required on salt.

Why I did this? Because `debconf-utils` is a problem. The package is at universe, so for SUSE Manager I had to add it to the client tools. But the package that creates it is `debconf` (which is on main), and FMPOV doing that is really dangerous.